### DISCUSSION TITLE
Enable showing/searching matches above 100 characters in the search bar

### DISCUSSION THREAD

**Discussion by robicina** [espanso/espanso#1268] [Feature requests and Ideas]
Is is possible to make search bar larger and use it to search through all the text I have saved in base.yml file. 
It is limited on 100 characters and it can't search show or search after 100 characters (inside search bar).

**Comment by federico-terzi** [comment]
Hey @robicina,

Thank you for the feedback! I'll mark it as a feature request :)

Cheers :)
